### DISCUSSION TITLE
Library is not working with RDS

### DISCUSSION THREAD

**Discussion by layshah72** [aws/aws-advanced-jdbc-wrapper#652] [Q&A]
I am trying to connect my RDS postgresql database using this wrapper but it is giving me error no pg_hba.conf entry for my server IP. RDS do not allow to modify entry in hba.conf file.

org.postgresql.util.PSQLException: FATAL: no pg_hba.conf entry for host "10.x.x.8", user "xxx_app_user", database "xxxdb", no encryption

When i am trying with previous version of driver class which was software.aws.rds.jdbc.postgresql.Driver it is working fine with same credentials.

**Comment by crystall-bitquill** [comment]
Hi @layshah72,

Could you please share what configuration parameters you're using and perhaps a bit of sample code to see how you're obtaining a connection?

Additionally, what version of RDS PostgreSQL are you using?

Thank you!

**Reply by layshah72** [reply]
I got the problem it was due to postgres driver version. I moved from  org.postgresql.postgresql 42.0.0 to 42.6.0 and it worked.. Thanks
